### DISCUSSION TITLE
Need help with Connector documentation for CoE Tool Kit

### DISCUSSION THREAD

**Discussion by shyjujohn** [microsoft/powerapps-tools#1170] [Q&A]
Tagging @manuelap-msft ..
Hi Team,

We are in the process of installing the CoE toolkit in our organization and during the review process the IT Security Team has raised a few question with regards to the Connectors and the permission level required for these connectors.
Can someone please throw some light on what these connectors does as part of CoE and their permission levels.

1. HTTPx  (Graph API – more info)
2. HTTP With Azure AD (Graph API - info)
3. Microsoft Teams  (purpose is for a channel)
4. Office 365 Groups ( more info)
5. Office 365 Outlook (more info)
6. Office 365 Users (more info)
7. PowerApps for Admins
8. PowerApps for Makers
9. Power Automate for Admins
10. Power Automate for Makers
11. Power Automate Management

Thank you!
Shyju John

**Comment by manuelap-msft** [comment]
This repo is no longer used or monitored by the CoE starter kit team. The new repo where you can raise issues is https://github.com/microsoft/coe-starter-kit/issues/new/choose 

We have updated all our docs to point to the new repo; please update your bookmarks if you still have the old link. Thank you.
